rclpy/rclpy/__init__.py

```diff
@@ -86,7 +86,8 @@ def init(
         else:
             signal_handler_options = SignalHandlerOptions.NO
     install_signal_handlers(signal_handler_options)
-    return context.init(args, domain_id=domain_id)
+    context.init(args, domain_id=domain_id)
+    return context
```

Comment:
The return type (`None`) and docstring should probably be updated.

Reply:
Yeah, good call.  I've done that now (although I also ended up doing a much more substantial update).